packages/react/src/components/DetailsList/DetailsColumn.test.tsx

```diff
@@ -184,11 +186,45 @@ describe('DetailsColumn', () => {
     expect(component.exists(`#${referenceId}`)).toBe(true);
   });
 
-  it('does not render invalid aria-describedby if custom DetailsHeader has onRenderColumnHeaderTooltip', () => {
+  it('does render invalid aria-describedby pointing to a custom tooltip not using ariaLabel', () => {
```

Comment:
Is this description accurate?

```suggestion
  it('renders valid aria-describedby pointing to a custom tooltip not using ariaLabel', () => {
```